packages/block-library/src/social-link/index.php

```diff
@@ -45,7 +45,7 @@ function render_block_core_social_link( $attributes, $content, $block ) {
 		)
 	);
 
-	return '<li ' . $wrapper_attributes . '><a href="' . esc_url( $url ) . '" aria-label="' . esc_attr( $label ) . '" ' . $attribute . ' class="wp-block-social-link-anchor"> ' . $icon . '</a></li>';
+	return '<li ' . $wrapper_attributes . '><a href="' . esc_url( $url ) . '" aria-label="' . esc_attr( $label ) . '" ' . $rel_target_attributes . ' class="wp-block-social-link-anchor"> ' . esc_html( $icon ) . '</a></li>';
```

Comment:
Escaping icon breaks the block output since the icons are SVGs.


![CleanShot 2022-01-14 at 11 36 59](https://user-images.githubusercontent.com/240569/149469325-ae00cba2-5657-41e5-a1ca-671619fbfabd.png)

Reply:
Thank you for catching this 😌 . I've been going through trying to remove instances where I'd inadvertently escaped SVGs.

Reply:
Fixed.